app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

```diff
@@ -20,8 +20,17 @@ expander:
       <p>If you have refugee status you'll usually be eligible for financial support to help you train.</p>
       <p>If you're in <a href="https://www.gov.uk/claim-asylum"> the process of seeking asylum</a>, check your immigration bail conditions to see if you have permission to study on a teacher training course in England. Even if you have permission, you're unlikely to be eligible for financial support.</p>
       <p>If you're granted asylum in the UK, you may be eligible for financial support to train to teach.</p>
+cta_arrow_link:
+  loans:
+    link_target: "https://www.gov.uk/student-finance"
+    link_text: "Check your eligibility and apply for student loans"
+  bursaries:
+    link_target: "/funding-and-support/scholarships-and-bursaries"
+    link_text: "Learn more about bursaries and scholarships for physics and languages"
+  find:
+    link_target: "https://find-teacher-training-courses.service.gov.uk"
+    link_text: "Find postgraduate teacher training and filter by salaried courses"
```

Comment:
Is this slightly risky that we are highlighting salaried courses so much through the green arrow? And sending people straight to Find rather than explain more about salaried courses first (they may well miss the first link in the section?)

Would it be better to remove the `work and earn a salary while you train` link and replace the green arrow link with a 'Find out more about salaried teacher training`? Particularly as we are updating the salaried page with more guidance re placements, etc.?